packages/babel-template/src/parse.ts

```diff
@@ -81,16 +80,16 @@ function placeholderVisitorHandler(
           "'.syntacticPlaceholders' is false.",
       );
     } else {
-      name = ((node: any).name: BabelNodeIdentifier).name;
+      name = node.name.name;
       state.isLegacyRef.value = false;
     }
   } else if (state.isLegacyRef.value === false || state.syntacticPlaceholders) {
     return;
   } else if (t.isIdentifier(node) || t.isJSXIdentifier(node)) {
-    name = ((node: any): BabelNodeIdentifier).name;
+    name = node.name;
```

Comment:
Here TS is better than flow at refining `node`

Reply:
it can be even better - if to first convert `@babel/types` and `@babel/traverse` packages (there are many types missing or to be improved comparing to definitely typed)